src/input_common/udp/client.cpp

```diff
@@ -63,6 +64,10 @@ class Socket {
     }
 
 private:
+    u32 GenerateRandomClientId() const {
+        std::random_device{}();
```

Comment:
```suggestion
        return std::random_device{}();
```
Missing `return` here.